pyproject.toml

```diff
@@ -36,11 +37,30 @@ pytest-clarity = "~0.3.0-alpha.0"
 pytest-timeout = "~1.3.4"
 asgi-lifespan = "~1.0.1"
 molten = "~1.0.2"
-sqlalchemy-stubs = "~0.4"
+
+sqlalchemy2-stubs = "^0.0.2-alpha.18"
+
+# extras packages are duplicated to dev-dependencies as they need for unit-tests
```

Comment:
Maybe use [pytest.importorskip](https://docs.pytest.org/en/latest/how-to/skipping.html#skipping-on-a-missing-import-dependency)?

Reply:
In this case some tests will  be just skipped in CI/CD.